test/measures/measures-specification-spec.js

```diff
@@ -28,7 +28,7 @@ function checkUrl(s) {
 // this will run once a day on travis
 if (process.env.TRAVIS_EVENT_TYPE === 'cron') {
   it('has valid specification links', function() {
-    this.timeout(8000); // 8 seconds timeout.
+    this.timeout(12000); // 8 seconds timeout.
```

Comment:
i would just delete the comment so it can't be wrong, it doesn't seem useful :p

Reply:
duh. fixed!